<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rooster</class>
 <widget class="QMainWindow" name="rooster">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>196</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>520</width>
    <height>196</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>520</width>
    <height>196</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Rooster</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>481</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="origin">
       <property name="minimumSize">
        <size>
         <width>287</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>287</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maxLength">
        <number>100</number>
       </property>
       <property name="placeholderText">
        <string>Start Address</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Ready in:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="pretime">
       <property name="minimumSize">
        <size>
         <width>95</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>95</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>This is the time it takes for you to get ready.</string>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="specialValueText">
        <string/>
       </property>
       <property name="suffix">
        <string> mins</string>
       </property>
       <property name="prefix">
        <string/>
       </property>
       <property name="maximum">
        <number>240</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
       <property name="value">
        <number>60</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>481</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="dest">
       <property name="minimumSize">
        <size>
         <width>287</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>287</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maxLength">
        <number>100</number>
       </property>
       <property name="placeholderText">
        <string>Destination Address</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Arrive at:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="arrivaltime">
       <property name="minimumSize">
        <size>
         <width>95</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>95</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>The time you wish to arrive at your destination.</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="specialValueText">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>481</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QSpinBox" name="beginchecking">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>95</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>There is no need to check traffic all night. Please set this to a reasonable value.</string>
       </property>
       <property name="specialValueText">
        <string/>
       </property>
       <property name="suffix">
        <string> mins</string>
       </property>
       <property name="maximum">
        <number>240</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
       <property name="value">
        <number>60</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string> before arrival time, begin checking traffic.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="alarmButton">
       <property name="text">
        <string>Set Alarm</string>
       </property>
       <property name="shortcut">
        <string>Return</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>520</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionFade_in_Alarm_Volume"/>
    <addaction name="actionChoose_Audio_File"/>
   </widget>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="statusTip">
    <string/>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionChoose_Audio_File">
   <property name="text">
    <string>Choose Audio File ...</string>
   </property>
  </action>
  <action name="actionFade_in_Alarm_Volume">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fade in Alarm Volume</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>origin</tabstop>
  <tabstop>dest</tabstop>
  <tabstop>pretime</tabstop>
  <tabstop>arrivaltime</tabstop>
  <tabstop>beginchecking</tabstop>
  <tabstop>alarmButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
